canonical-websites/www.ubuntu.com
# Demo of new primary/secondary/global navigation

## Done

- Built new navs (**design and copy are NOT finalised**), which includes:
  - Primary nav ([copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y), [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2644))
  - Download dropdown([copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y), [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2647))
  - Secondary nav ([copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y), [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2648))
  - Global nav ([copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y), [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2643))

- Rebased code with master branch so I could have access to the /download/core pages

## TODO

- Make primary nav stick to the top of the page (except when at the top, because that's where the global nav lives)
- Add animation to dropdowns
- Any semblance of responsive design

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Play around with the new nav
